### PR TITLE
Feature/style the app

### PR DESCRIPTION
**Pull request by rulles13**
Start refactoring to use components

### CODE REVIEW

File: app/components/DepartureRow.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import DepartureTimeHr from "./DepartureTimeHr"
+import Headsign from "./Headsign"
+import ShortName from "./ShortName"
+
+function DepartureRow(props: any) {
+
+    const {depart} = props
```
Comment (heleneruelle) [inline app/components/DepartureRow.tsx]: departure is more english

File: app/components/DepartureTimeHr.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function DepartureTimeHr(props: { depart: any }) {
+
+    const {depart} = props
+
+    const {departureDateTime} = depart
+    const departTime = new Date(departureDateTime).toLocaleTimeString('fr-EU', { hour: '2-digit', minute: '2-digit' })
```
Comment (heleneruelle) [inline app/components/DepartureTimeHr.tsx]: departure

File: app/pages/SmartDisplay.tsx [hunk 3]
```diff
@@ -16,34 +15,17 @@ function SmartDisplay() {
...
 
-                )
-                : (<p>error no departure</p>)
-            }
-        </div>
+    return (
+      <div className="main-container">
+        <Header stopDetails={stopDetails}/> 
+        <ul>
+          {departures.map((depart: any, index: number) => (
```
Comment (heleneruelle) [inline app/pages/SmartDisplay.tsx]: error handling

**Comment by rulles13** [comment]
image

**Comment by heleneruelle** [comment]
Nice !

File: app/components/DepartureRow.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            <Headsign departure={departure} />
+            <DepartureTimeHr departure={departure} />
+        </div>
+    )
+}
+
+export default DepartureRow
+
+// {
+//     departures && departures.length
```
Comment (heleneruelle) [inline app/components/DepartureRow.tsx:23]: remove comment if you don't use them

File: app/components/DepartureTimeHr.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+function DepartureTimeHr(props: { departure: any }) {
+
+    const {departure} = props
+
+    const {departureDateTime} = departure
+    const departureTime = new Date(departureDateTime).toLocaleTimeString('fr-EU', { hour: '2-digit', minute: '2-digit' })
```
Comment (heleneruelle) [inline app/components/DepartureTimeHr.tsx:6]: In this doc you will find everything thats avail as otpions for Intl formatting : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat
(keep it somewhere to have ot close when needed)

In this case you can also just use `timeStyle: "short'` but what you did is also great !

**Review by heleneruelle — APPROVED** [review]

File: app/components/DepartureTimeHr.tsx (hunk 5 above)
Comment (rulles13) [reply app/components/DepartureTimeHr.tsx:6]: Thanks for the link. I save it in my favourite.

**Pull request merged**